src/sentry/integrations/gitlab/integration.py

```diff
@@ -266,17 +267,17 @@ def build_integration(self, state):
         secret = sha1_text(''.join([hostname, state['installation_data']['client_id']]))
 
         integration = {
-            'name': group['name'],
+            'name': group['full_name'],
             # Splice the gitlab host and project together to
             # act as unique link between a gitlab instance, group + sentry.
             # This value is embedded then in the webook token that we
             # give to gitlab to allow us to find the integration a hook came
             # from.
-            'external_id': u'{}:{}'.format(hostname, group['path']),
+            'external_id': u'{}:{}'.format(hostname, group['full_path']),
```

Comment:
Thank you. I could imagine a scenario where only doing the `group['path']` would cause issues and it not be unique with subgroups.